src/Microsoft.IdentityModel.Tokens/JsonWebKeySet.cs

```diff
@@ -97,6 +99,22 @@ public JsonWebKeySet(string json)
         [JsonIgnore]
         public bool SkipUnresolvedJsonWebKeys { get; set; } = DefaultSkipUnresolvedJsonWebKeys;
 
+        /// <summary>
+        /// The original <see cref="string"/> used to create this instance.
```

Comment:
I don't think a cref string is useful here. Maybe something like this?

```suggestion
        /// The original string used to create this instance if a string was provided.
```